analysis/magefire/src/CHANGELOG.tsx

```diff
@@ -4,6 +4,12 @@ import { Adoraci, Sharrq } from 'CONTRIBUTORS';
 import { SpellLink } from 'interface';
 
 export default [
+  change(date(2021, 12, 8), <>Added a suggestion for expired <SpellLink id={SPELLS.SUN_KINGS_BLESSING.id} />.</>, Sharrq),
```

Comment:
Seems like a bunch of fire changes are in this - probably related to a PR that has been merged, so try and merge shadowlands branch into this PR so we're sure what's being added by this PR